colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,351 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bpGidixAICM1"
+      },
+      "source": [
+        "# Use an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jZC69sSMdX4j"
+      },
+      "source": [
+        "When you need to select a particular Artifact, indicate the artifact you want to use with the `use_artifact` method. This returns a run object. In the proceeding code snippet specifies an artifact called `my_first_artifact` with the alias `latest`:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7Ix_ciolJICe"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "u60Orwg5sYuI"
+      },
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Create a new Artifact version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When you need add a new file, you can use the `artifact.add_file` method."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"new_file\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds a new .csv file called `new_file` to the `my_first_artifact` Artifact. \n",
+        "\n",
+        "Optionally, you can choose to add an entire directory to an Artifact:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact.add_dir(local_path=\"/content/sample_data\", name=\"new_directory\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds an entire folder with several .csv files to your Artifact.\n",
+        "\n",
+        "To save your changes, you'll need to `log` them, which is covered in the next section."
```

Comment:
why is log covered in a separate section? I'm not seeing this separate section anywhere?

Reply:
That's an artifact from a previous section- I removed it.